src/applications/gi/selectors/calculator.js

```diff
@@ -1023,9 +1023,7 @@ export const getCalculatedBenefits = createSelector(
       };
     }
 
-    if (institution.yr &&
-      derived.tier === 1.0 &&
-      institution.yellowRibbonPrograms.length > 0) {
```

Comment:
The institution end point provides a boolean that indicates if an institution has a yellow ribbon program. Adjusted the drop down behavior so the app continues to use this attribute to determine when to display the yellow ribbon question.